cypress.config.js

```diff
@@ -1,12 +1,29 @@
 import { defineConfig } from 'cypress'
+import { rmdir } from 'fs'
 
 export default defineConfig({
   fixturesFolder: 'tests/e2e/fixtures',
   screenshotsFolder: 'tests/e2e/screenshots',
   videosFolder: 'tests/e2e/videos',
+  downloadsFolder: 'tests/e2e/downloads',
   e2e: {
     // eslint-disable-next-line no-unused-vars
-    setupNodeEvents(on, config) {},
+    setupNodeEvents(on, config) {
```

Comment:
What is this for? Maybe some docs.

Reply:
Its so we can access node events in the cypress tests. In this case its to clean up the csv files downloaded from the sample report test so they don't linger.

Documented in [be96327](https://github.com/sanger/traction-ui/pull/2370/commits/be963278088eea3cb21330f9fd17dbc4d71cb13d)